src/main/java/no/nav/familie/inntektsmelding/imdialog/tjenester/InntektsmeldingMapper.java

```diff
@@ -110,25 +110,35 @@ private static List<SendInntektsmeldingRequestDto.Refusjon> mapRefusjonerTilDto(
         if (entitet.getMånedRefusjon() != null) {
             refusjoner.add(new SendInntektsmeldingRequestDto.Refusjon(entitet.getStartDato(), entitet.getMånedRefusjon()));
         }
+        // Frontend forventer at opphørsdato mappes til en liste der fom = første dag uten refusjon, må derfor legge på en dag.
```

Comment:
Som diskutert på slack - i utgangspunktet synes jeg det er mer fornuftig å trekke fra en dag i mapper til xml, og opprettholde mappingen som vi har tenkt i ft-inntektsmelding. Det gjør det mindre komplisert i ft-inntektsmelding. 

Som det ble nevnt i slack diskusjonen så er ulempen at vi må huske at xmlen forventer siste dag i refusjon, og vi operer med refusjonsdato som opphørsdato når vi senere skal endre/erstatte xml med noe annet

Reply:
Lukker denne da vi ble enige på møtet om å opprettholde mapping opphørsdato minus 1 dag for å få siste dag i refusjonsperioden, slik som xml forventer